external/pico_sdk_import.cmake

```diff
@@ -40,25 +40,26 @@ if (NOT PICO_SDK_PATH)
         if (PICO_SDK_FETCH_FROM_GIT_PATH)
             get_filename_component(FETCHCONTENT_BASE_DIR "${PICO_SDK_FETCH_FROM_GIT_PATH}" REALPATH BASE_DIR "${CMAKE_SOURCE_DIR}")
         endif ()
-        # GIT_SUBMODULES_RECURSE was added in 3.17
```

Comment:
Does the new version of the code still need to do something different if we're using an older version of CMake which doesn't support `GIT_SUBMODULES_RECURSE` ?

Reply:
> now that TinyUSB doesn't have any submodules

Good point, I hadn't considered that! But might be better to err on the side of caution though, just in case we add any other submodules in future which _do_ have submodules of their own? :thinking: